spec/scheduling-tasks.md

```diff
@@ -238,28 +324,32 @@ A <dfn>task handle</dfn> is a [=struct=] with the following [=struct/items=]:
   [=scheduler task queue/tasks=] is [=list/empty=].
 </div>
 
-
-### Scheduling Tasks ### {#sec-scheduler-alg-scheduling-tasks}
+### Scheduling Tasks and Continuations ### {#sec-scheduler-alg-scheduling-tasks-and-continuations}
 
 <div algorithm>
   To <dfn>schedule a postTask task</dfn> for {{Scheduler}} |scheduler| given a
   {{SchedulerPostTaskCallback}} |callback| and {{SchedulerPostTaskOptions}} |options|:
 
   1. Let |result| be [=a new promise=].
-  1. Let |signal| be |options|["{{SchedulerPostTaskOptions/signal}}"] if
-     |options|["{{SchedulerPostTaskOptions/signal}}"] [=map/exists=], or otherwise null.
+  1. Let |state| be the result of [=computing the scheduling state from options=] given |scheduler|,
```

Comment:
Nit: "compute the scheduling state from options"

Reply:
Any particular reason? It's common IIUC to use a different conjugation than the algorithm to make things more readable, and bikeshed handles the linking auto-magically (i.e. this links to "compute the ..." but is grammatically correct).

Reply:
oh, cool.  I was just using find in doc.